eclipse-theia/theia
# plugin: add support for `title` property in `InputBoxOptions`

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10030.

The pull-request adds support for the `title` property in `InputBoxOptions` which is used to give a title to quick-input menus.
The change also includes styling updates to the quick-input to give the input the same styling as the rest of the application's input fields, and also sets the `title` background similarly to VS Code.

https://user-images.githubusercontent.com/40359487/159915091-655c72c5-28bc-4163-a228-da2747bea9ee.mp4

#### How to test


1. include the following test extension: [vscode-input-0.0.1.zip](https://github.com/eclipse-theia/theia/files/8341337/vscode-input-0.0.1.zip)
2. start the application, and execute the `hello world` command
3. a quick-input should be triggered which has a `title` property
4. confirm the quick-input work
5. using different themes confirm that quick-inputs are properly styled

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
